kotsadm/web/src/scss/components/troubleshoot/SupportBundleAnalysis.scss

```diff
@@ -305,6 +333,83 @@
   transform: translateX(-2px);
 }
 
+.Timeline--wrapper {
+  position: relative;
+  width: 100%;
+  padding: 0 0 0 15px;
+  border-left: 2px solid #C4C4C4;
+
+  .section {
+    position: relative
+  }
+
+  .section:before {
+    content: "";
+    display: block;
+    height: 20px;
+    border-left: 2px solid #c4c4c4;
+    border-bottom: 2px solid #c4c4c4;
+    border-radius: 0 0 0 100px;
```

Comment:
Same here, since the only value is `border-bottom-left-radius` it should be written as such.